src/Containers/containerize/ContainerizeCommand.cs

```diff
@@ -185,6 +185,18 @@ internal class ContainerizeCommand : CliRootCommand
 
     internal CliOption<string> ContainerUserOption { get; } = new("--container-user") { Description = "User to run the container as." };
 
+    internal CliOption<bool> GenerateLabelsOption { get; } = new("--generate-labels")
+    {
+        Description = "If true, the tooling may create labels on the generated images.",
```

Comment:
Am I reading it correctly that the new `GenerateDigestLabelWithoutGenerateLabels` message is localized but these option descriptions are not?

Reply:
yes that's correct - the CLI is only used by the .NET Framework pathway here, and this pathway is on the way to removal over the medium term. The only part of the existing containerization pathway that _must_ be .NET 7+ is the tarball creation, and we can do that by making a small standalone .NET 7 executable. The rest of the containerization workflow could be done in a more well-factored task and take advantage of the existing helpers/patterns for MSBuild localization. My WIP branch towards this world is [here](https://github.com/baronfel/sdk/tree/resolve-base-image-in-msbuild).